README.md

```diff
@@ -271,6 +271,8 @@ Caught a mistake or want to contribute to the documentation? [Edit this page on
  - [elm-feather](https://github.com/1602/elm-feather) - Feather icons for Elm applications
  - [react-feather](https://github.com/carmelopullara/react-feather) - Feather icons as React components
  - [vue-feather-icon](https://github.com/mage3k/vue-feather-icon) - Feather icons as Vue components
+ - [sketch-feather](https://github.com/odmln/sketch-feather) - Feather icons as a Sketch library
```

Comment:
Awesome! Maybe we should move this link above `vue-feather-icon` to keep the list in alphabetical order?

Reply:
Agreed!